behat_suites.yml

```diff
@@ -35,10 +35,8 @@ adminui:
                 - EzSystems\EzPlatformAdminUi\Behat\BusinessContext\DashboardContext
                 - EzSystems\RepositoryForms\Behat\Context\ContentTypeContext
                 - EzSystems\RepositoryForms\Behat\Context\FieldTypeFormContext
-                
-adminuimodules:
-    suites:
-        default:
+
+        adminuimodules:
```

Comment:
This should be a profile, it's expected in admin-ui-modules: https://github.com/ezsystems/ezplatform-admin-ui-modules/blob/master/.travis.yml#L12

Reply:
Fixed by https://github.com/ezsystems/ezplatform-admin-ui-modules/pull/182